backend/src/server/context.ts

```diff
@@ -38,4 +39,15 @@ export const getUser = (context: Context): dbUser => {
   throw new Error('No user given in context!')
 }
 
+export const getClientRequestTime = (context: Context): Date => {
+  // console.log('context:', context)
+  if (context.clientRequestTime) {
+    logger.info(`context with clientRequestTime=${context.clientRequestTime}`)
+    return new Date(context.clientRequestTime)
```

Comment:
This is not working. With the example of Tokyo.
```
conext.clientRequestTime = 2022-11-01T04:49:32.648+09:00
```
You get the server time as result:
```
2022-10-31T19:49:32.648Z
```

Reply:
We could just cut off the timezone offset and the test suite should pass again. But I fear we run in far more difficult problems, if we do so.

Reply:
to process the testcases successfully the settings of the clientRequestTime have to fullfill some prerequisits:
- if the timezone offset > 0 the time have to be > the timezone offset to avoid switching the date back to the same server date in UTC
- if the timezone offset < 0 the time have to be < the timezone offset to avoid switching the date forward to the same server date in UTC